openquake/hazardlib/tests/gsim/chiou_youngs_2008_swiss_test.py

```diff
@@ -25,7 +25,7 @@ class ChiouYoungs2008SWISS01TestCase(BaseGSIMTestCase):
 
     def test_std_total(self):
         self.check('CY08Swiss/cy_2008_phis_ss_embeded.csv',
-                   max_discrep_percentage=1.50)
+                   max_discrep_percentage=2.00)
```

Comment:
No, for the time being. These tables were obtained via several iterations to represent the embedded adjustments, so    the raise to 2% tolerance is alright.